examples/datacenters/aws-k8s/aws-lb-controller/Pulumi.yaml

```diff
@@ -0,0 +1,3 @@
+name: k8s-service
+runtime: nodejs
+description: A minimal DigitalOcean TypeScript Pulumi program
```

Comment:
Not for digitalocean actually. Just for k8s.

Reply:
Good call, you can tell I copy pasted these base pulumi modules from the DO datacenter 😛